SmartDeviceLink-iOS/SmartDeviceLink/SDLRPCRequestFactory.h

```diff
@@ -93,6 +94,7 @@
 + (SDLAlert *)buildAlertWithAlertText1:(NSString *)alertText1 alertText2:(NSString *)alertText2 duration:(NSNumber *)duration correlationID:(NSNumber *)correlationID;
 //*****
 
++(SDLAlertManeuver*) buildAlertManeuverwithTTSchunks:(NSMutableArray *)ttsChunks softButtons:(NSMutableArray*) softButtons correlationID:(NSNumber*) correlationID;
```

Comment:
The formatting on this line, as well as lines 201, 203 and 209 need to be correct. There is a missing space before, as well as an additional space after, the return type. This is to keep consistent with the rest of the file/project.